### ISSUE TITLE
Send to owners: exclude Exodus?

### ISSUE THREAD

**Issue by dexX7** [mastercoin-MSC/spec#299]
While testing STO I noticed Exodus is not explicitly excluded from the receivers.

Is there any reason *not* to exclude Exodus here?

In my opinion it's not an active participant in the system to begin with, but it also skews (reg-) testing.

It's also not allowed to send tokens to Exodus, so it would only be consistent to exclude Exodus.

**Comment by zathras-crypto** [comment]
@dexX7 just saw your PR come through - sorry I thought I'd posted this earlier, seems I never clicked comment d'oh!

I think this discussion has been had already mate, I'd have to go hunting to cite.  IIRC the first draft of code Michael put together did have Exodus excluded, but that was reversed.

> Is there any reason not to exclude Exodus here?

The tl:dr; there is no real reason *to* exclude Exodus - it's just an address like any other.  We don't exclude the issuer of other tokens, so why put a special rule in for propID 1 & 2?  Exodus is an address that owns n % of MSC, so if a MSC SendToOwners is broadcast, I see no reason to skew the results by excluding n% from the totals - eg if someone owns 4% of MSC, they get 4% of an MSC STO; excluding Exodus would skew that.

> It's also not allowed to send tokens to Exodus, so it would only be consistent to exclude Exodus.

That's a technical limitation (since the Exodus address is used as a marker, there's ambiguity on whether a send was really meant for Exodus) so safer to just to exclude Exodus from reference address identification.  Since STO doesn't use reference address identification (there is no recipient specified) this limitation doesn't apply here.

Long story short it was about treating MSC like any other property (I'm anti special rules for any properties even MSC).  Exclude Exodus and we should exclude every property issuer - instead I believe we came to the conclusion that excluding the sender would be appropriate - in most cases the sender *is* the issuer and in the case that some party other than the issuer wants to issue dividends on a property, why should the issuer not get their share?

Maybe a new v1 transaction could have the option to specifically exclude the issuer from getting a cut of the goodies (so in the case of an MSC/TMSC STO that would exclude Exodus) but I'm not sure where that lands on the map.

Thanks :)
Z

**Comment by dexX7** [comment]
While it seems reasonable to exclude senders of STO, I somehow disagree to treat Exodus similar to regular issuers, because Exodus acts much more like a passive black hole instead of an active participant.

I share your sentiment regarding "less artificial rules" though, and will retreat. Thanks for the clarification! :)
